backend/src/apiserver/template/argo_template.go

```diff
@@ -89,15 +90,38 @@ func (t *Argo) ScheduledWorkflow(apiJob *api.Job) (*scheduledworkflow.ScheduledW
 	if err != nil {
 		return nil, util.Wrap(err, "Create job failed")
 	}
+	//
+
+	var pipelineID *string
```

Comment:
I see your point on this, I will update it will require some refactoring. Will continue to push it tomorrow and hopefully have something up then. Will put prio on this during the week so hope I can get some time to push it again.

Reply:
I guess we could add the raw manifest incase the pipelineID and pipelineVersionID is empty as an alternative.

Reply:
@Bobgy

Reply:
Agree, we can store the raw manifest (v2 pipeline spec) into swf CR, similar to how users can create a run using raw manifest.